src/Parser/Core/Modules/DamageMitigation/Armor.js

```diff
@@ -0,0 +1,24 @@
+import Analyzer from 'Parser/Core/Analyzer';
+
+class Armor extends Analyzer {
```

Comment:
Why did you make this class instead of using StatTracker?

Reply:
Using the armor from the damage event is more accurate. Could update the statTracker to be updated based on that, but it might be confusing that the implementation is so different to the other stats. Also it would not update until a damage event, so if for some reason you tried to get the armor for something else (idk what that would be) between hits it might not return the correct value.